packages/create-webpack-app/README.md

```diff
@@ -0,0 +1,41 @@
+<div>
+    <a href="https://github.com/webpack/webpack-cli">
+        <img width="200" height="200" src="https://webpack.js.org/assets/icon-square-big.svg">
+    </a>
+</div>
+
+# create-webpack-app CLI
+
+## About
+
+- `create-webpack-app` is a cli tool that enables developers to quickly scaffold a new webpack project. It provides a flexible set of commands for developers to increase speed when setting up a custom webpack project. webpack CLI addresses these needs by providing a set of tools to improve the setup of custom webpack configuration.
+- It is also going to support several front end frameworks and libraries like React, Angular, Vue, Svelte, etc.
+- Webpack Loader and Plugin scaffolding is also supported.
+
+## How to install
+
+```bash
+npm install --save-dev @webpack-cli/create-webpack-app
```

Comment:
Installing it locally should not be required, `npx create-react-app my-app` should work directly.